taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -87,13 +87,20 @@ def __init__(self, projectid, sampleid, **kwargs):
         self.reportpath = getattr(self, 'reportpath', None)
         self.force = getattr(self, 'force', False)
         self.stage_only = getattr(self, 'stage_only', False)
-        # only set an attribute for uppnexid if it's actually given or in the db
+
+        #Fetch parameters from command line; if not available try calling database
+	try:
+	    self.projectname = getattr(self, 'projectname')
+	except AttributeError:
+            try:
+                self.projectname = getattr(db.project_entry(db.dbcon(), projectid), 'name')
```

Comment:
you need to do `self.projectname = db.project_entry(db.dbcon(), projectid)['name']` in order to throw a `KeyError`